ckanext/ckanext-sixodp_ui/ckanext/sixodp_ui/templates/package/resource_read.html

```diff
@@ -118,8 +121,8 @@ <h2 class="page-subheading">
                 {% if res.description %}
                   {{ h.render_markdown(h.get_translated(res, 'description')) }}
                 {% endif %}
-                  {% if not res.description and c.package.notes %}
-                  <h3>{{ _('From the dataset abstract') }}</h3>
+                {% if not res.description and c.package.notes %}
+                  <h2>{{ _('From the dataset abstract') }}</h2>
```

Comment:
"Datan kuvaus" heading is now h2 which is the same as the resource date heading (if it is rendered).
Btw, "Lisätietoja" is level h3. Is that ok?
![image](https://user-images.githubusercontent.com/63100253/92869972-0fcb7b80-f40c-11ea-8bfa-307ed520fd67.png)

Reply:
I think its fine accessibility wise, page subheading probably should have some actual heading but it will be fixed later.